scripts/api/api.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+def create_data_file(api_url, data_file_data):
+    request_url = api_url + "day"
+    headers = {"content-type": "application/json"}
+    r = requests.post(
+        request_url,
+        headers=headers,
+        data=data_file_data,
+    )
+    print(r)
```

Comment:
I don't think it should print. Maybe a `log.info`?

Reply:
https://github.com/AMP-SCZ/dpimport/commit/17a12ba14df570de31c11435e4ade8b837bbb3a0